lib/matrix.test.js

```diff
@@ -33,3 +33,21 @@ test('product test',()=>{
     ]
   });
 });
+
+test('transpose test', () => {
+  let m = new Matrix(3, 3);
```

Comment:
I wonder if a non 3x3 Matrix would be a better test?

Reply:
So, we know that 3x3 may work with these tests. If we want to check other kinds of matrices, should we just create other tests with different names or we need to do something specific because we're testing the same method/function in different ways (hope this is a good place to ask this).